types/wlr_output_layout.c

```diff
@@ -8,14 +8,14 @@
 #include <assert.h>
 
 struct wlr_output_layout_state {
-	struct wlr_box *_box;
```

Comment:
I put an underscore here to emphasize that you should never read this, but only use the getter.

Reply:
Oh, okay, I'll add a comment too then.